201-machine-learning-dependencies-role-assignment/azuredeploy.parameters.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
let's add parameter values here that will test all the resources in the template (i.e. change the false to true on the flags)...  you'll need to add more to the prereq template as well...

Reply:
done